src/dialog-editor/components/field/fieldComponent.ts

```diff
@@ -34,6 +36,25 @@ class FieldController {
   public editDialogModal(tab: number, box: number, field: number) {
     this.DialogEditorModal.showModal(tab, box, field);
   }
+
+  /**
+   * Remove Field
+   * @memberof FieldController
+   * @function remmoveField
+   * @param {number} tabId is an index of tab, where the box is placed
+   * @param {number} boxId is an index of box, where the field is placed
+   * @param {number} fieldId is an index of field
+   */
+  public removeField(tabId: number, boxId: number, fieldId: number) {
+    _.remove(
```

Comment:
This construction begs for undefined. Can you rewrite it perhaps to this?
```Typescript
class getGroupByBoxId {
  //..
  private static getTabById(tabs, tabId) {
    return tabs[tabId];
  }
  private static getGroupByBoxId(groups, boxId) {
    return groups[boxId];
  }

  private static getDialogFields(tabs, tabId, boxId) {
    const tab = FieldController.getTabById(tabs, tabId);
    const group =  tab && FieldController.getGroupByBoxId(tab.dialog_groups, boxId);
    return group && group.dialog_fields;
  }

  public removeField(tabId: number, boxId: number, fieldId: number) {
    const fields = FieldController.getDialogFields(this.DialogEditor.getDialogTabs(), tabId, boxId);
    fields && _.remove(fields, (field: any) => field.position === fieldId);
  }
}
```

Reply:
This class and methods are long standing, I do not want to start making changes in there, especially if unrelated to this particular PR.